test/suites/drift/suite_test.go

```diff
@@ -279,8 +273,14 @@ var _ = Describe("Drift", func() {
 			By("drifting the nodepool")
 			nodePool.Spec.Template.Annotations = lo.Assign(nodePool.Spec.Template.Annotations, map[string]string{"test-annotation": "drift"})
 			env.ExpectUpdated(nodePool)
+
+			By("waiting for disruption to start")
+			env.EventuallyExpectTaintedNodeCount(">", 0)
+
+			By("checking max 3 out of 5 nodes to be disrupted at any time")
 			env.ConsistentlyExpectDisruptionsUntilNoneLeft(5, 3, 15*time.Minute)
```

Comment:
Without the wait, `ConsistentlyExpectDisruptionsUntilNoneLeft` could blow past before the disruption even starts. Making this wait part of  `ConsistentlyExpectDisruptionsUntilNoneLeft` may be better, but taking a conservative route for now. Other tests use `EventuallyExpectDrifted` before, and should not be affected.